### PR TITLE
feat: implement changelog for installer (pt.1)

### LINKED ISSUE #104: [REQUEST] Installer Changelog
**Is your feature request related to a problem? Please describe.**

Maybe add a Changelog for the installer somewhere in settings (for example if you click on the version number it opens a window) Would be nice to see what new features, fixes have been implemented

**Describe the feature you'd like to see implemented**

**Additional context**

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by FoxtrotSierra6829**
Fixes #104 

## Summary of Changes
Add installer changelog in settings menu

## Screenshots
grafik

## Additional context
Automated displaying after an update is still to be implemented #119

Discord username (if different from GitHub):
Foxtrot Sierra#6420

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ This repo contains the installer prototype for FlyByWire Simulations projects.
...
-Please make sure you have: 
+Please make sure you have:
 - NodeJS 14 - [Homepage](https://nodejs.org/en/)
 
 ## How to open
 
 First install the dependencies
 ```shell script
-npm install
+npm ci
```
Comment (ZigTag) [inline README.md:16]: Undo this change

**Review by ZigTag — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (FoxtrotSierra6829) [reply README.md:16]: It's better for compatibility, to install the already preconfigured dependencies initially via ``npm ci`` (that's why you commit package-lock.json) and only use ``npm install`` when you intend to update or add dependencies, but if you want me to, sure.

**Review by ZigTag — APPROVED** [review]
LGTM
